test/src/balance.test.js

```diff
@@ -11,17 +11,24 @@ contract('balance', function ([sender, receiver]) {
     });
   });
 
-  describe('difference', function () {
+  describe('balance tracker', function () {
```

Comment:
We should also add tests for `get`, and for how `get` and `delta` interact (i.e. the fact that `get` sets a new checkpoint).